README.md

```diff
@@ -47,27 +47,55 @@ Default: `local.net/nfs`
 
 Default: `local-nfs`
 
-### `hostPath`
+### `provisionerVolume`
 
-If this is empty, no local storage will be used, making this completely emphemeral.
+There are a number of modes you can use. These are the available options.
 
-Default: `/srv/nfs-provisioner`
+| mode               | description                                                                                                                                                                                       | available settings                                                          |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| hostPath (default) | A hostPath volume mounts a file or directory from the host node’s filesystem into your pod  https://kubernetes.io/docs/concepts/storage/volumes/#hostpath                                         | * path: "/directory/location/on/host"    (defaults to /srv/nfs-provisioner) |
```

Comment:
By the way, the lists (starting with `*`) don't seem to render in the table. Are you aware of any workaround?

Reply:
see #7